doc/association-rules.rst

```diff
@@ -0,0 +1,79 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+#### Inputs
+
+-  **Data**
+
+Data set
+
+#### Outputs
+
+-  **Matching Data**
+
+Data instances matching the criteria.
+
+Description
+-----------
+
+This widget runs the original `Agrawal's algorithm <http://cs-people.bu.edu/evimaria/cs565/advances.pdf>`__ for the induction of
```

Comment:
> This add-on contains only the documented algorithms.

I understood this as 'it contains Agrawal's algorithm' since it is documented. Should I just copy-paste the first sentence of the doc then?